packages/vite-plugin-cloudflare/src/workers-configs.ts

```diff
@@ -0,0 +1,353 @@
+import assert from 'node:assert';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { unstable_readConfig } from 'wrangler';
+import { name } from '../package.json';
+import type { AssetsOnlyConfig, WorkerConfig } from './plugin-config';
+import type { Optional } from './utils';
+import type { Unstable_Config } from 'wrangler';
+
+type RawWorkerConfig = Unstable_Config;
+
+export namespace WorkersConfigurations {
```

Comment:
Isn't `namespace` sort of deprecated in TS now?

Reply:
oof... that's embarrassing 🙈  yeah they seem to be, somehow that completely flew under my radar (I haven't used them a lot)

they seemed like such a nice way to scope similar concepts here and avoid naming collisions 😓 

I'll have to go back to the drawing board and use plain unspaced types 👍

Reply:
Although there are other people who disagree... https://www.reddit.com/r/typescript/comments/1d3zpow/comment/l6bn086/

Reply:
namespaces removed 🙂  https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/112/commits/e9710a3d9f54ad5c2f149b9103c31c7503651b8d